appveyor.yml

```diff
@@ -19,17 +19,23 @@
 os: Visual Studio 2015
 
 only_commits:
-  # Skip commits not related to Python or C++
+  # Skip commits not related to Python, C++ or Rust
   files:
     - appveyor.yml
     - ci/
     - cpp/
     - python/
+    - rust/
```

Comment:
At some point we will want to selectively disable some matrix builds based on the modified files, as we already do for Travis-CI. The `ci/travis_detect_changes.py` script does that, it can be extended for AppVeyor. The following logic (from CPython) is supposed to be able to detect changed files on AppVeyor PRs:
https://github.com/python/cpython/blob/master/.github/appveyor.yml#L12-L26

Builds already pile up quickly due to concurrency limitations on AppVeyor, ideally we should not make things worse.

Reply:
Yea, it's pretty annoying that all cpp build would re-trigger on changes to the PR.  I'll create a JIRA for it.  It's probably something I could take a look at.

Reply:
[JIRA](https://issues.apache.org/jira/browse/ARROW-2516?filter=-2)

Reply:
Thank you!